HealthcareSystem/Backend/Controller/RoomAndEquipment/EquipmentController.cs

```diff
@@ -14,8 +14,8 @@ namespace Controller.RoomAndEquipment
 {
    public class EquipmentController
 {
```

Comment:
Fix indents , Ctrl+K, Ctrl+D, which indents the whole file

Reply:
Solved in 227dd73d685d299759a4b4d3dbf857be00385e68 as far as I can see.